Services/Repository/ConfigRepositoryInterface.php

```diff
@@ -19,4 +19,6 @@ public function save($key, $value, $type = null, bool $locked = false, bool $for
     public function removeByKey($key);
 
     public function getConfigurationValue($key);
+
+    public function findByKey($key);
```

Comment:
getConfigurationValue($key) already exist. Then why do we need another function named findByKey